mod/homepage/lib/javascript/script_homepage_carousel.js.coffee

```diff
@@ -64,10 +107,48 @@ activateIntroTab = (tab)->
   active_panel.find('.carousel').carousel()
   active_panel.find('.carousel-item').first().addClass 'active'
 
+getNumberElements = () -> 
+  values = []
+  $('h1.font-weight-normal_count-ele').each -> 
+    values.push( $(this) )
+    controlAnimate($(this))
+  values
+  
+isScrolledIntoView = (elem) ->
+  docViewTop = $(window).scrollTop();
+  docViewBottom = docViewTop + $(window).height();
+  elemTop = $(elem).offset().top;
+  elemBottom = elemTop + $(elem).height();
+  ((elemBottom <= docViewBottom) && (elemTop >= docViewTop));
+
+runAnimation = (elem) ->
+  options = { useEasing: true, useGrouping: true, separator: ',', decimal: '.', };
+  demo = new CountUp($(elem).attr('id'),  0, parseInt($(elem).text()), 0, 3.5, options);
```

Comment:
we need to remove the delimiter `number_with_delimiter(number, delimiter: ",")` from `hompeage_numbers.rb' ` in order make this `parseInt($(elem).text())` work properly. number is already rendered with comma, so this parseInt won't work well. 
eg. 
output of parseInt("3434,5454")  will be 3434

Reply:
also why `demo` ?